### PR TITLE
User/chbhatt/adding notes on blank html, unverified removal, Accountinfo and SignInAccountInfo

### PR DESCRIPTION
**Pull request by chbhatt**
This PR does following:
1. Adds detailed notes on redirect_uri behavior in popup mode. This is explain when blank.html is apt for this scenario.
2. Adds an entry for "unverified" in troubleshooting section which points to the document with info on how to fix it.
3. Updates `AccountInfo` and `SignInAccountInfo` structure documentation to indicate which fields can be empty and which cannot be.

### CODE REVIEW

File: docs/quick-authentication-how-to.md [hunk 1]
```diff
@@ -339,6 +339,17 @@ To also enable logging for MSAL.js, add the [`logMsalEvents`](./quick-authentica
         src="https://edge-auth.microsoft.com/js/ms_auth_client.min.js?autoLogEvents=1&logMsalEvents=4"
         ></script>
 ```
+## Notes on Redirect URI for Single-Page application for popup mode
+
+If ux_mode is set to `'popup'` (which is default), then on clicking sign-in button or sign-in prompt, the credential / consent dialog opens up in a popup.
```
Comment (christian-fortini) [inline docs/quick-authentication-how-to.md:344]: Suggest: **the** default

File: docs/quick-authentication-how-to.md [hunk 2]
```diff
@@ -339,6 +339,17 @@ To also enable logging for MSAL.js, add the [`logMsalEvents`](./quick-authentica
...
+
```
Comment (christian-fortini) [inline docs/quick-authentication-how-to.md:345]: Suggest: **the** sign-in button or sign-in prompt

File: docs/quick-authentication-how-to.md [hunk 3]
```diff
@@ -339,6 +339,17 @@ To also enable logging for MSAL.js, add the [`logMsalEvents`](./quick-authentica
...
+The popup will be redirected to redirect URI [registered above](#register-your-application) and specified in quick authentication configuration using [login_uri](./quick-authentication-reference.md#data-type-initconfiguration) or [data-login_uri](./quick-authentication-reference.md#specifying-the-quick-authentication-initialization-parameters) by MSA server, once the credential entry / consenting process is completed on the server side. The redirection URL will contain a fragment with info needed by quick authentication to complete the sign-in process in client side.
```
Comment (christian-fortini) [inline docs/quick-authentication-how-to.md:346]: Not sure I understand what "by MSA server" refers to.

Comment (christian-fortini) [reply docs/quick-authentication-how-to.md:346]: Oh. I see. By the time I got to "by MSA server" I had lost the flow of the sentence... :)

Suggest: will be redirected by MSA server to redirect URI...

File: docs/quick-authentication-how-to.md [hunk 4]
```diff
@@ -339,6 +339,17 @@ To also enable logging for MSAL.js, add the [`logMsalEvents`](./quick-authentica
...
+
+Quick authentication **needs** that the domain of the page which included our library is the same as domain of the redirect URI specified.
```
Comment (christian-fortini) [inline docs/quick-authentication-how-to.md:348]: Needs the domain ... to be the same as...

File: docs/quick-authentication-how-to.md [hunk 5]
```diff
@@ -339,6 +339,17 @@ To also enable logging for MSAL.js, add the [`logMsalEvents`](./quick-authentica
...
+
+After server redirection, once quick authentication detects that the domain is same as the domain which loaded its Javascript library, it will close the popup and complete the sign-in process in client side.
```
Comment (christian-fortini) [inline docs/quick-authentication-how-to.md:350]: Suggest: in client side -> on the client side

File: docs/quick-authentication-reference.md [hunk 6]
```diff
@@ -146,33 +146,33 @@ This is an optional string value which can be configured using any of the follow
...
-| `homeAccountId` | User's [home account identifier](https://azuread.github.io/microsoft-authentication-library-for-js/ref/modules/_azure_msal_common.html#accountinfo) which is used by MSAL |
+| Property | Description | More info |
+|----------|---------|---------------|
+| `fullName` | User's full name | Can be empty |
+| `surname` | User's surname | Can be empty |
+| `givenName` | User's given name | Can be empty |
+| `username` | User's email or phone number | Not empty |
+| `email` | User's email address | Not empty |
+| `photoUrl` | base64-encoded dataURI representing the user's avatar. If the user has set no avatar, its value will be `null`| Can be empty |
+| `id` | an ID that is unique across all Microsoft accounts. This id can be used for some Microsoft Graph APIs (e.g., [User GET](https://learn.microsoft.com/en-us/graph/api/user-get?view=graph-rest-1.0&tabs=http)). | Not empty |
```
Comment (christian-fortini) [inline docs/quick-authentication-reference.md:157]: Suggest: Never empty

**Review by christian-fortini — CHANGES_REQUESTED** [review]

File: docs/quick-authentication-how-to.md (hunk 1 above)
Comment (chbhatt) [reply docs/quick-authentication-how-to.md]: Fixed.

File: docs/quick-authentication-how-to.md (hunk 5 above)
Comment (chbhatt) [reply docs/quick-authentication-how-to.md]: Fixed.

File: docs/quick-authentication-reference.md (hunk 6 above)
Comment (chbhatt) [reply docs/quick-authentication-reference.md]: Fixed.

**Pull request merged**